packages/eas-cli/src/graphql/queries/UpdateQuery.ts

```diff
@@ -112,4 +135,142 @@ export const UpdateQuery = {
         .toPromise()
     );
   },
+  async viewUpdateGroupAsync({ groupId }: { groupId: string }): Promise<ViewUpdatesByGroupQuery> {
+    return await withErrorHandlingAsync(
+      graphqlClient
+        .query<ViewUpdatesByGroupQuery, ViewUpdatesByGroupQueryVariables>(
+          gql`
+            query ViewUpdatesByGroup($groupId: ID!) {
+              updatesByGroup(group: $groupId) {
+                id
+                group
+                runtimeVersion
+                platform
+                message
+                createdAt
+                manifestFragment
+                actor {
```

Comment:
it looks like `getActorDisplayName` requires `__typename`, so we may want to explicitly query that here and update the type of `getActorDisplayName` to match. I think overall there's a lot of work that can be done around types though, so if we want to punt on that to a follow-up PR we can.

Reply:
yeah! I mentioned above I'm planning to follow this up with a PR to introduce fragments etc
https://linear.app/expo/issue/ENG-6202/refactor-improve-gql-typing-around-updates